explorer/parse_and_execute/BUILD

```diff
@@ -25,13 +25,14 @@ cc_library(
     ],
 )
 
-cc_test(
+# This is included in //explorer:explorer_tests.
```

Comment:
Why the comment here but not elsewhere? I think consistency in either direction would be fine, but right now it seems the odd one out.